urbanairship-react-native/package.json

```diff
@@ -15,29 +14,57 @@
   ],
   "devDependencies": {
     "@babel/core": "7.5.4",
-    "@babel/plugin-transform-flow-strip-types": "^7.4.4",
-    "@babel/runtime": "7.5.4",
+    "@react-native-community/bob": "^0.15.1",
```

Comment:
Something to be aware of for future typescript/react work – this is a recent update to bob from a few days ago that fixes a bug that would make it return with a successful exit code when tsc reported syntax errors. We should make sure we don't use versions prior to this.